common/scala/src/main/scala/whisk/common/Config.scala

```diff
@@ -42,113 +42,120 @@ import scala.util.Try
  * @param optionalProperties a Set of optional properties which may or not be defined.
  * @param env an optional environment to read from (defaults to sys.env).
  */
-class Config(
-    requiredProperties: Map[String, String],
-    optionalProperties: Set[String] = Set())(
-        env: Map[String, String] = sys.env)(
-            implicit logging: Logging) {
-
-    private val settings = getProperties().toMap.filter {
-        case (k, v) => requiredProperties.contains(k) ||
-            (optionalProperties.contains(k) && v != null)
-    }
+class Config(requiredProperties: Map[String, String], optionalProperties: Set[String] = Set())(
+    env: Map[String, String] = sys.env)(implicit logging: Logging) {
```

Comment:
seems like things are at 2 space indent - can wrapping be the same?

Reply:
For wrapping this is quite handy to distinguish the class body from the class header. I'd keep that as is (it's the standard config)